docs/platform_introduction/annotations.md

```diff
@@ -0,0 +1,20 @@
+# Annotations
+One important aspect of *osparc* is its adherence to **FAIR** (**F**indable **A**ccessible **I**nteroperable **R**eusable) data and code-based artifacts (e.g. models, simulations, data processing modules). To that end, we enable several different features to promote FAIR principles. 
```

Comment:
perhaps a reference to the nature article? https://www.nature.com/articles/sdata201618

Reply:
done!